ml-agents-envs/mlagents/envs/timers.py

```diff
@@ -0,0 +1,172 @@
+# # Unity ML-Agents Toolkit
```

Comment:
Better suggestions on where this should live?

Reply:
This seems good to me for the short term.